### PR TITLE
Hook the CLI up to the TxPool Server

### PR DESCRIPTION
**Pull request by SamWilsn**

### CODE REVIEW

File: turbo-txpool/src/tx.rs [hunk 1]
```diff
@@ -92,7 +112,167 @@ mod tests {
     use super::*;
 
     #[test]
-    fn tx_decode() -> Result<(), Error> {
+    fn encode_deploy() {
+        let input = vec![
```
Comment (lightclient) [inline turbo-txpool/src/tx.rs:116]: Thoughts on using hex strings here (in tests in general) instead?

**Review by lightclient — APPROVED** [review]
Looks good. Guess we'll wait to make sure CI passes then merge.

File: turbo-txpool/src/tx.rs (hunk 1 above)
Comment (SamWilsn) [reply turbo-txpool/src/tx.rs:116]: Something like [`hex-literal`](https://crates.io/crates/hex-literal)?

Comment (lightclient) [reply turbo-txpool/src/tx.rs:116]: Yeah that looks perfect. I've found that sometimes it's really nice to be able to easily copy the test cases and actually use with in the program.

**Pull request merged**